### PR TITLE
Adds holofans to CE locker and Red/Gamma ERT Engineer backpacks

### PR DESCRIPTION
**Pull request by MrMagolor**
**What does this PR do:**
Include a small to medium description of what your PR changes. Feel free to give it a little flourish if you want to show off a new feature more completely.

This PR adds an ATMOS Holofan Projector to the CE's locker, as well as giving one to the backpacks of Red and Gamma ERT Engineers.

**Images of sprite/map changes (IF APPLICABLE):**
If you did not make a map or sprite edit, you may leave this section blank. You may include a gif of your feature if you want

**Changelog:**
*Remove this line, and appropriate fields from the changelog*
:cl:
tweak: The CE's locker and Red/Gamma ERT Engineer backpacks now have ATMOS Holofan Projectors.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 	new /obj/item/rpd(src)
 	new /obj/item/reagent_containers/food/drinks/mug/ce(src)
 	new /obj/item/organ/internal/cyberimp/eyes/meson(src)
+	new /obj/item/holosign_creator/atmos
```
Comment (Couls) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Missing (src)

**Comment by datlo** [comment]
On one hand, its nice that atmos tech have their own unique thing. On the other, this often results in a CE just pillaging atmos lockers since they have access. I think this is fine once the code is fixed.

**Comment by TDSSS** [comment]
Reading the changelog I first thought all ert were going to get it, not just engies.

**Comment by variableundefined** [comment]
@MrMagolor PM me on discord or ping me here when deconflicted please.

**Comment by MrMagolor** [comment]
@variableundefined seems to be all good now

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 2]
```diff
@@ -1,149 +1,150 @@
-/obj/structure/closet/secure_closet/engineering_chief
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: What went wrong; this concerns me.

Comment (Couls) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: Oh, looks like they used the website to resolve conflicts and it screwed with the format

Comment (Fox-McCloud) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm]: @MrMagolor pleased fix this.

**Pull request closed without merging**